samples/client/petstore/cpprest/api/StoreApi.h

```diff
@@ -35,7 +35,8 @@ namespace api {
 
 using namespace io::swagger::client::model;
 
-class  StoreApi
+
```

Comment:
I think we should avoid spurious whitespace changes in generated code as it's just diff noise.

Reply:
Now that we always generate the ABCs it's no longer an issue.